src/hooks/useFetch.ts

```diff
@@ -1,22 +1,29 @@
-import React from 'react';
+import { useState, useEffect } from 'react';
+import fetch from '@/helperFunctions/fetch';
 
-const useFetch = (url: string, options: object = {}) => {
-  const [response, setResponse] = React.useState(null);
-  const [error, setError] = React.useState(null);
-  const [isLoading, setIsLoading] = React.useState(false);
-  React.useEffect(() => {
+const useFetch = (url: string) => {
```

Comment:
If this is a reusable utility, I think a way to provide options would be good to have. So it can be used for any type of usecase.